packages/fluentui/accessibility/src/behaviors/Box/dropdownClearIndicatorBehavior.ts

```diff
@@ -0,0 +1,24 @@
+import { Accessibility } from '../../types';
+
+/**
+ * @description
+ * Indicator is usually only visual representation and therefore is hidden from screen readers, unless 'aria-label' property is provided.
+ * Adds attribute 'tabIndex=-1' to 'root' slot if 'search' property is false. Does not set the attribute otherwise.
+ *
+ * @specification
+ * Adds role='button'.
+ */
+const dropdownClearIndicatorBehavior: Accessibility<DropdownClearIndicatorBehaviorProps> = props => ({
+  attributes: {
+    root: {
+      role: 'button',
```

Comment:
what do you think about the adding condition(!props.search) for the role as well?
I would propose it, because even element has no tabIndex, but if it is button for example JAWS can navigate there with Virtual cursor
I did testing with this example:
https://codesandbox.io/s/fluent-ui-example-2plsz?file=/example.js